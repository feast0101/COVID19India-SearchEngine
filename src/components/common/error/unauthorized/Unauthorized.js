import React from 'react';

import '../error.css';

const Unauthorized = () => {
  return (
    <div className='error-page'>
      <h3>You are unauthorized to access this application.</h3>
    </div>
  );
};

export default Unauthorized;

