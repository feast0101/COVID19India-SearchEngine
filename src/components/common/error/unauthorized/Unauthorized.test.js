import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import Unauthorized from "./Unauthorized";

describe("Unauthorized", () => {
  it("runs snapshot test", () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(<Unauthorized/>);
    expect(result).toMatchSnapshot();
  });
});

