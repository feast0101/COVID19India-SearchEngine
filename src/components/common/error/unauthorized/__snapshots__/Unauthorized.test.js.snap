// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unauthorized runs snapshot test 1`] = `
<div
  className="error-page"
>
  <h3>
    You are unauthorized to access this application.
  </h3>
</div>
`;
